packages/ui/__tests__/components/organisms/HeaderContainer.test.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {TouchableOpacity, View} from 'react-native';
+import {shallow} from 'enzyme';
+import {HeaderContainer, Icon} from '@axelor/aos-mobile-ui';
+import {getGlobalStyles} from '../../tools';
+
+describe('HeaderContainer Component', () => {
+  const props = {
+    fixedItems: <View testID="fixedItems" />,
+    children: <View testID="children" />,
+    chipComponent: <View testID="chipComponent" />,
+  };
+
+  it('should render without crashing', () => {
+    const wrapper = shallow(<HeaderContainer {...props} />);
+
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  it('should render fixedItems, children and chipComponent', () => {
+    const wrapper = shallow(<HeaderContainer {...props} />);
+
+    expect(wrapper.find('[testID="fixedItems"]').exists()).toBe(true);
+    expect(wrapper.find('[testID="children"]').exists()).toBe(true);
+    expect(wrapper.find('[testID="chipComponent"]').exists()).toBe(true);
+  });
+
+  it('should not render children if expandableFilter is false', () => {
+    const wrapper = shallow(
+      <HeaderContainer {...props} expandableFilter={false} />,
+    );
+
+    expect(wrapper.find('[testID="children"]').exists()).toBe(false);
+  });
+
+  it('should not render children if forceHideByDefault is true', () => {
+    const wrapper = shallow(<HeaderContainer {...props} forceHideByDefault />);
+
+    expect(wrapper.find('[testID="children"]').exists()).toBe(false);
+  });
```

Comment:
Need to add a test to check config management too with a spyOn on useConfig. It has already been done in another component